mvnxparser/mvnxStreamReader/mvnxStreamReader.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright: (C) 2017 iCub Facility
+ * Author: Diego Ferigo
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ *
+ */
+
+/**
+ * @file mvnxStreamReader.h
+ * @brief Validate and parse efficiently mvnx files
+ * @author Diego Ferigo
+ * @date 18/04/2017
+ */
+#ifndef MVNX_STREAM_READER_H
+#define MVNX_STREAM_READER_H
+
+#include "xmlDataContainers.h"
+#include "xmlStreamReader.h"
+#include <QXmlStreamReader>
+#include <iostream>
+#include <memory>
+#include <unordered_map>
+#include <vector>
+
+namespace xmlstream {
+namespace mvnx {
+
+// Simple container of the mvnx parsing configuration.
+typedef std::unordered_map<std::string, bool> mvnxConf;
+
+class mvnxStreamReader : public xmlstream::xmlStreamReader {
```

Comment:
I like classes to start capital case.
Given that MVNX is an acronym, I would call  the class
`MVNXStreamReader `.

This also for the other classes